config/routes.rb

```diff
@@ -4,6 +4,10 @@
 
   resources :categories, only: [:index]
   resources :items
+  resource :cart, only: [:show, :delete, :edit, :update]
+  get '/:category' => 'categories#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.